frontend/src/components/auth/components/sign-in-form/sign-in-form.tsx

```diff
@@ -19,6 +24,12 @@ const SignInForm: FC<Props> = ({ onSubmit }) => {
       validationSchema: masterSignInValidationSchema,
     });
 
+  const [isClicked, setIsClicked] = useState(false);
+
+  const clickHandler = (): void => {
+    setIsClicked(!isClicked);
+  };
```

Comment:
```suggestion
```
we just want to add some style on hover. That it. Only css should be change
Example - https://jabber-my.space/sign-in
(hover on the button)